packages/local-cli/runIOS/findMatchingSimulator.js

```diff
@@ -31,7 +31,7 @@ function findMatchingSimulator(simulators, simulatorString) {
   const parsedSimulatorName = simulatorString
     ? simulatorString.match(/(.*)? (?:\((.*)?\))?/)
     : [];
-  if (parsedSimulatorName[2] !== undefined) {
+  if (parsedSimulatorName && parsedSimulatorName[2] !== undefined) {
```

Comment:
This is interesting. 

On line 31, we make sure it's either a matched value or an array - it shouldn't be "undefined" at all.

Could you provide more details when this would be null, or causing a crash?

Reply:
Ah, it confused me:
```
simulatorString
    ? simulatorString.match(/(.*)? (?:\((.*)?\))?/)
    : [];
```
Thought we check on `simulator.match` instead xd